create-presigned-download-url/src/main/java/no/unit/nva/download/publication/file/publication/RestPublicationService.java

```diff
@@ -0,0 +1,101 @@
+package no.unit.nva.download.publication.file.publication;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.mikael.urlbuilder.UrlBuilder;
+import no.unit.nva.download.publication.file.publication.exception.NoResponseException;
+import no.unit.nva.download.publication.file.publication.exception.NotFoundException;
+import no.unit.nva.model.Publication;
+
+import nva.commons.exceptions.ApiGatewayException;
+import nva.commons.utils.Environment;
+import nva.commons.utils.JacocoGenerated;
+import nva.commons.utils.JsonUtils;
+import org.apache.http.HttpHeaders;
+
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.util.UUID;
+
+import static org.apache.http.HttpStatus.SC_NOT_FOUND;
+
+public class RestPublicationService {
```

Comment:
It's starting to look like we need a nva-publication-client project to reuse this client across projects.

Reply:
I agree, I think alot of projects will need to access publications. Task added: NP-867